_data/simple-icons.json

```
@@ -8317,9 +8317,23 @@
 		"guidelines": "https://github.com/jupyter/design"
 	},
 	{
-		"title": "Just Eat",
-		"hex": "F36D00",
-		"source": "https://justeattakeaway.com/newsroom/en-WW/media_kits/"
+		"title": "Just Eat Takeaway.com",
+		"hex": "FF8000",
+		"source": "https://brand-box.marketing.just-eat.com",
+		"guidelines": "https://brand-box.marketing.just-eat.com",
+		"aliases": {
+			"loc": {
+				"de-AT": "Lieferando.at",
+				"de-DE": "Lieferando.de",
+				"en-AU": "Menulog",
+				"en-CA": "SkipTheDishes",
+				"en-US": "Grubhub",
+				"he-IL": "10bis",
+				"nl-NL": "Thuisbezorgd.nl",
+				"pl-PL": "Pyszne.pl",
+				"sk-SK": "Bistro.sk"
```

```suggestion
				"de-AT": "Lieferando.at",
				"de-DE": "Lieferando.de",
				"en-AU": "Menulog",
				"en-CA": "SkipTheDishes",
				"en-US": "Grubhub",
				"fr-CA": "Skip",
				"he-IL": "10bis",
				"nl-NL": "Thuisbezorgd.nl",
				"pl-PL": "Pyszne.pl",
				"sk-SK": "Bistro.sk"
```

Here, in Québec, it is only called `Skip`.

---

In fact, I think it called `Skip` everywhere in Canada now according to https://en.wikipedia.org/wiki/Skip_(delivery_service)